src/Symfony/Bundle/FrameworkBundle/Tests/Translation/TranslatorTest.php

```diff
@@ -104,34 +104,6 @@ public function testTransWithCachingWithInvalidLocale()
         $translator->trans('foo');
     }
 
-    public function testLoadResourcesWithCaching()
```

Comment:
I used `ResourceCheckerConfigCacheFactory`, the problems occurs when we want to work with the debug mode, is there a way to pass debug mode for `ResourceCheckerConfigCacheFactory` like the [old one](https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/Translation/Translator.php#L499)  ?